scripts/geo_exclusions.json

```diff
@@ -0,0 +1,6 @@
+{
+    "exclusions":
+    {
+        "2024-03-24": "ak,ca"
```

Comment:
This is Sunday; I guess it's supposed to indicate the week, rather than the forecast date? Either way, it didn't actually get picked up